api/spec/v1.yaml

```diff
@@ -1036,6 +1036,30 @@ paths:
                 $ref: '#/components/schemas/TokenHolderList'
         <<: *common_error_responses
 
+  /{runtime}/evm_nfts/{address}:
+    get:
+      summary: |
+        Returns the list of non-fungible token (NFT) instances of an EVM (ERC-721, ...) token.
+        This endpoint does not verify that `address` is actually an EVM token; if it is not, it will simply return an empty list.
+      parameters:
```

Comment:
Would it be possible to have an optional parameter for specifying the instance ID, for getting data about a specific NFT instance?

(Or is there / will there be a separate API endpoint for that?)

Reply:
let's add another route with /id at the end

Reply:
That is also fine

Reply:
oh or have we more commonly done it by parameter?

Reply:
No, I think having it in the path is perfect

Reply:
adding in #538

Reply:
gonna resolve here. let's discuss there